README.md

```diff
@@ -4,79 +4,202 @@
 # PyCon Taiwan Blog
 
 ## Getting Started
+### Environment Setup
 
-### Prerequisites
-* [Python 3.10](https://www.python.org/downloads/)
-* [pipenv](https://pipenv.pypa.io/en/latest/)
+1. Fork & Clone [pycontw-blog](https://github.com/pycontw/pycontw-blog)
 
+    For those who already know how to fork the repository, you can skip to the next step.
 
-### Environment Setup
+    <details>
+    <summary> Click me</summary>
+
+    1.1 Navigate to [pycontw-blog](https://github.com/pycontw/pycontw-blog) and press the `Fork` button
+    on the top right corner.
+    <img src="./content/images/step_1_fork_repo.png" />
+
+    1.2 Press `Create fork`
+    <img src="./content/images/step_2_create_fork.png" />
+
+    1.3 Copy the URL of the forked repo
+    <img src="./content/images/step_3_copy_url.png" />
+    </details>
+
+2. Clone the forked repo
+
+    ```bash
+    git clone --recursive <YOUR_URL_HERE>
+    ```
+
+3. Install [miniconda](https://docs.anaconda.com/free/miniconda/#quick-command-line-install)
```

Comment:
I would suggest we don't bind our python release with a specific release except for CPython

Reply:
Regarding how python should be installed, I would suggest this talk by TP https://www.youtube.com/watch?v=6Nl0IYkU0hm, even though my personal preference is pyenv haha. AFAIK, `pipenv` works well with `pyenv` so python version ideally won't be an issue in this setup. The remaining issue is how we install `pipenv` and maybe `invoke` as well. I would suggest using https://github.com/pypa/pipx

Reply:
https://www.youtube.com/watch?v=6Nl0IYkU0hU

my bad. not sure what's happening on the link

Reply:
I do get your point. Yep, we can keep `minconda` stuff. But I might suggest some rewording. We probably could say `miniconda` is one of the ways we install python instead of list it as a required step.

e.g., ~~3. Install [miniconda](https://docs.anaconda.com/free/miniconda/#quick-command-line-install)~~ -> 3. Install Python
You can download Python from the official site or follow these steps to install it with miniconda